bitops.h

```diff
@@ -0,0 +1,8 @@
+#ifndef __BITOPS_H__
+#define __BITOPS_H__
+
+#define   SET_BIT(n, b) (n |=  (1 << b) )
+#define CLEAR_BIT(n, b) (n &= ~(1 << b) )
```

Comment:
For consistency and to avoid macro expansion related errors:

#define CLR_BIT(n, b) ((n) &= ~(1 << (b)))

Also, please add INV_BIT.